src/pages/Pinned.js

```diff
@@ -0,0 +1,58 @@
+import Card from '../components/Card';
+import Navigation from '../components/Navigation';
+import DeleteModal from '../components/modals/DeleteModal';
+import styled from 'styled-components';
+
+export default function Pinned({
+  cards,
+  onDeleteConfirm,
+  onTrashClick,
+  onKeepConfirm,
+  showModal,
+  onPinClick,
+}) {
+  return (
+    <PositionWrapper>
+      <StyledList role="list">
+        {cards.map(card => {
+          if (card.isPinned === true)
+            return (
+              <li key={card._id}>
+                <Card
+                  _id={card._id}
+                  question={card.question}
+                  answer={card.answer}
+                  onTrashClick={onTrashClick}
+                  onPinClick={onPinClick}
+                  isPinned={card.isPinned}
+                />
+              </li>
+            );
+          else return [];
+        })}
+
+        {showModal && (
```

Comment:
when there is a modal, the content outside should not be reachable for the user, important also for accessibility - just as a suggestion for a further userstory